src/config.js

```diff
@@ -18,135 +18,103 @@ class Config {
     fs.ensureFileSync(this.configPath)
 
     if (fs.readFileSync(this.configPath).toString() === '') {
-      this.devices = {}
-      this.save()
+      this.reset()
     }
 
-    this.devices = require(this.configPath)
+    this.config = require(this.configPath)
   }
 
-  // Save configuration to file system.
-  save () {
-    fs.writeFileSync(this.configPath, JSON.stringify(this.devices, null, 2))
-    return true
-  }
-
-  // Get the argument after -d or --deviceName
-  // Or return the first device name
-  getDefaultDeviceName () {
-    for (let index = 0; index < process.argv.length; index++) {
-      let arg = process.argv[index]
-      if ((arg === '-d') || (arg === '--deviceName')) {
-        return process.argv[index + 1]
-      }
-    }
+  // Reset the configuration
+  reset () {
+    this.config = Object.create(null)
+    this.clear()
+    this.persist()
+  }
 
-    return Object.keys(this.devices)[0]
+  // Save configuration to file system.
+  persist () {
+    fs.writeFileSync(this.configPath, this.toJSON())
   }
 
-  // Return config related to device name.
-  getDevice (deviceName) {
-    if (deviceName == null) { deviceName = this.getDefaultDeviceName() }
+  // Transform the config to a JSON string
+  toJSON () {
+    return JSON.stringify(this.config, null, 2)
+  }
 
-    if (this.devices[deviceName] != null) {
-      return this.devices[deviceName]
-    } else if (Object.keys(this.devices).length === 0) {
-      return {} // No device configured
-    } else {
-      log.error(`Device not set locally: ${deviceName}`)
-      throw new Error(`Device not set locally: ${deviceName}`)
-    }
+  // Get the path on the local file system of the synchronized folder
+  getSyncPath () {
+    return this.config.path
```

Comment:
I'm not sure whether es getters/setters are supported... Would it make sense to use them?